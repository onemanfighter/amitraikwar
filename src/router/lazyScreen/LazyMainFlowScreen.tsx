import { lazy } from 'react';
import { LazyComponentProvider } from '@providers';

const MainScreen = lazy(() => import('@screens/mainFlow/MainScreen'));

export const LazyMainScreen = () => {
  return (
    <LazyComponentProvider>
      <MainScreen />
    </LazyComponentProvider>
  );
};
