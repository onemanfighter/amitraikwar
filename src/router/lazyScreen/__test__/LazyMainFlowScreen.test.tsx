import { render } from '@testing-library/react';
import { LazyMainScreen } from '../LazyMainFlowScreen';

jest.mock('react-social-icons', () => ({
  SocialIcon: ({ url }: { url: string }) => <div>{url}</div>,
}));

describe('LazyMainFlowScreen', () => {
  it('should render lazy about screen', async () => {
    const { container } = render(<LazyMainScreen />);

    expect(container).toMatchSnapshot();
  });
});
