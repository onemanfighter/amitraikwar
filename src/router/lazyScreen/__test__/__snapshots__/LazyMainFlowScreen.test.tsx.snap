// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LazyMainFlowScreen should render lazy about screen 1`] = `<div />`;
